src/utils/highlighter.js

```diff
@@ -0,0 +1,194 @@
+// hljs.registerLanguage('sas', require('highlight.js/lib/languages/sas'));
+hljs.registerLanguage('scala', require('highlight.js/lib/languages/scala'));
+// hljs.registerLanguage('scheme', require('highlight.js/lib/languages/scheme'));
+// hljs.registerLanguage('scilab', require('highlight.js/lib/languages/scilab'));
+hljs.registerLanguage('scss', require('highlight.js/lib/languages/scss'));
+hljs.registerLanguage('shell', require('highlight.js/lib/languages/shell'));
+// hljs.registerLanguage('smali', require('highlight.js/lib/languages/smali'));
+hljs.registerLanguage('smalltalk', require('highlight.js/lib/languages/smalltalk'));
+// hljs.registerLanguage('sml', require('highlight.js/lib/languages/sml'));
+// hljs.registerLanguage('sqf', require('highlight.js/lib/languages/sqf'));
+hljs.registerLanguage('sql', require('highlight.js/lib/languages/sql'));
+// hljs.registerLanguage('stan', require('highlight.js/lib/languages/stan'));
+// hljs.registerLanguage('stata', require('highlight.js/lib/languages/stata'));
+// hljs.registerLanguage('step21', require('highlight.js/lib/languages/step21'));
+// hljs.registerLanguage('stylus', require('highlight.js/lib/languages/stylus'));
+// hljs.registerLanguage('subunit', require('highlight.js/lib/languages/subunit'));
+hljs.registerLanguage('swift', require('highlight.js/lib/languages/swift'));
+// hljs.registerLanguage('taggerscript', require('highlight.js/lib/languages/taggerscript'));
+hljs.registerLanguage('yaml', require('highlight.js/lib/languages/yaml'));
+// hljs.registerLanguage('tap', require('highlight.js/lib/languages/tap'));
+hljs.registerLanguage('tcl', require('highlight.js/lib/languages/tcl'));
+// hljs.registerLanguage('tex', require('highlight.js/lib/languages/tex'));
+// hljs.registerLanguage('thrift', require('highlight.js/lib/languages/thrift'));
+// hljs.registerLanguage('tp', require('highlight.js/lib/languages/tp'));
+// hljs.registerLanguage('twig', require('highlight.js/lib/languages/twig'));
+hljs.registerLanguage('typescript', require('highlight.js/lib/languages/typescript'));
+// hljs.registerLanguage('vala', require('highlight.js/lib/languages/vala'));
+hljs.registerLanguage('vbnet', require('highlight.js/lib/languages/vbnet'));
+hljs.registerLanguage('vbscript', require('highlight.js/lib/languages/vbscript'));
+hljs.registerLanguage('vbscript-html', require('highlight.js/lib/languages/vbscript-html'));
+// hljs.registerLanguage('verilog', require('highlight.js/lib/languages/verilog'));
+// hljs.registerLanguage('vhdl', require('highlight.js/lib/languages/vhdl'));
+hljs.registerLanguage('vim', require('highlight.js/lib/languages/vim'));
+// hljs.registerLanguage('x86asm', require('highlight.js/lib/languages/x86asm'));
+// hljs.registerLanguage('xl', require('highlight.js/lib/languages/xl'));
+// hljs.registerLanguage('xquery', require('highlight.js/lib/languages/xquery'));
+hljs.registerLanguage('zephir', require('highlight.js/lib/languages/zephir'));
```

Comment:
I think we can store keys of languages we want to support in an array constant then register them in a loop.
It would be easier to see which languages are enabled that way.

Reply:
Technically, we can. But webpack won't evaluate the `require` code then and consequently include all languages in the bundle. To avoid that, we'll have to configure webpack to remove the unneeded languages..

Reply:
OK. Never mind then.

Reply:
Done.